src/md/tekwill/jf4/homework/Exercise6.java

```diff
@@ -10,7 +14,21 @@
      * 2.138888888888889
      */
 
-    public static void main(String[] args) {
-
+    public static void main(String[] args)
+    {
+        try
+        {
+            ScriptEngineManager manager = new ScriptEngineManager();
+            ScriptEngine engine = manager.getEngineByName("JavaScript");
+            System.out.println(engine.eval(args[0]));
```

Comment:
we do java ))

Reply:
Wait... so this coursr is not about Javascript? Aren't Java and js the same thing? :P